### PR TITLE
fix: ai-generate-evaluation-question-with-"evaluation"-field

### PR DESCRIPTION
**Pull request by cbx6664**

### CODE REVIEW

File: IntelliEdu-application-service/src/main/java/com/team6/intellieduapplicationservice/controller/QuestionController.java [hunk 1]
```diff
@@ -190,20 +191,40 @@ public SseEmitter aiGenerateQuestionSse(@RequestBody AiGenerateQuestionRequest a
...
         String userMessage = getGenerateQuestionUserMessage(application, questionNumber, optionNumber);
-        ChatCompletionRequest chatCompletionRequest = aiManager.generalStreamRequest(GENERATE_QUESTION_SYSTEM_MESSAGE, userMessage, 0.5);
 
-        //建立 sse 连接对象，0表示永不超时
-        SseEmitter emitter = new SseEmitter(0L);
 
-        // 处理 AI 生成题目的请求
-        CompletableFuture<String> future = new CompletableFuture<>();
+        //测评类应用
+        if (application.getType() == AppType.EVALUATION.getCode()) {
```
Comment (cpuoverload) [inline IntelliEdu-application-service/src/main/java/com/team6/intellieduapplicationservice/controller/QuestionController.java:199]: duplicate code

Comment (cbx6664) [reply IntelliEdu-application-service/src/main/java/com/team6/intellieduapplicationservice/controller/QuestionController.java:199]: ok resolved

File: IntelliEdu-application-service/src/main/java/com/team6/intellieduapplicationservice/controller/QuestionController.java [hunk 2]
```diff
@@ -163,38 +164,42 @@ ApiResponse<Question> getQuestionById(@PathVariable Long id) {
...
+        userMessage.append("Application category: ").append(AppType.fromCode(application.getType()).getDescription() + " type").append("\n");
+        userMessage.append("Number of questions to generate: ").append(questionNumber);
+        userMessage.append("Number of options per question: ").append(optionNumber);
         return userMessage.toString();
     }
 
 
     @GetMapping("/ai_generate/sse")
-    public SseEmitter aiGenerateQuestionSse(AiGenerateQuestionRequest aiGenerateQuestionRequest) {
+    public SseEmitter aiGenerateQuestionSse(Long appId, int questionNumber, int optionNumber) {
```
Comment (cpuoverload) [inline IntelliEdu-application-service/src/main/java/com/team6/intellieduapplicationservice/controller/QuestionController.java:177]: int should be Integer?

Comment (cbx6664) [reply IntelliEdu-application-service/src/main/java/com/team6/intellieduapplicationservice/controller/QuestionController.java:177]: sure

**Pull request merged**